src/pages/Dashboard.tsx

```diff
@@ -23,4 +12,4 @@ function Dashboard(props: any) {
     );
 }
 
-export default IsLoadingHOC(Dashboard, "Loading...");
+export default Dashboard;
```

Comment:
Something I try to do (there are weird expectations as to why you can't do it always) is to never export default but do named exports instead. The main reason being is that Intelli-sense will be much smart in its ability to import your exports.